src/input_cp2k_global.F

```diff
@@ -730,6 +733,22 @@ SUBROUTINE create_grid_section(section)
       CALL section_add_keyword(section, keyword)
       CALL keyword_release(keyword)
 
+      CALL keyword_create(keyword, __LOCATION__, name="APPLY_CUTOFF", &
+                          description="When enabled the cpu and gpu backends "// &
+                          " apply a spherical cutoff on the top of the cube. "// &
+                          " There is a performance penalty using it in "// &
+                          "combination with the cpu backend but it is on by "// &
+                          "default for the regtests", default_l_val=.TRUE., &
+                          lone_keyword_l_val=.TRUE.)
+      CALL section_add_keyword(section, keyword)
+      CALL keyword_release(keyword)
+
+      CALL keyword_create(keyword, __LOCATION__, name="QUEUE_LENGTH", &
+                          description="Number of gaussians that the gpu backend treats in one round", &
```

Comment:
gpu -> hybrid

Reply:
corrected, thanks.

Reply:
Are you sure? I still read "gpu backend".

Reply:
Corrected. My apologies I thought I corrected it.